### ISSUE TITLE
DifferentFile options

### ISSUE THREAD

**Issue by katalystkorner** [easydigitaldownloads/easy-digital-downloads#4606]
Right now, if you have two files, you can attach each of them to all options or only one. The problem is, more diversity is needed. For example: File A. File B are uploaded. 
option 1: file a only
option 2: file b only
option 3: file a & b.
Please let me know if there is something I am not seeing, but I cannot get this to work in the current system and it should definitely be added! I have already had complaints from this from potential buyers. Thank you!

**Comment by cklosowski** [comment]
Are you think something like checkboxes? So if you have 3 price variations, with the current platform it can belong to all or one. Adding your concept would mean it could be any combination?

**Comment by katalystkorner** [comment]
Correct. Right now, for a file, you can say all or one. I want either a check box option to say which of the options, or the ability to select multiple in the drop down with the ctrl button. 
File a: option1, option3
File b:option2, option3

**Comment by chriscct7** [comment]
I think this might be better suited as an extension. Something like this could get really messy very quickly with a large number of files uploaded unless we swapped the variations to use a vertical metabox style tabbed interface like WooCommerce where each variation can have set options

**Comment by cklosowski** [comment]
I disagree, this is pretty basic functionality that I think we should be able to provide. Thoughts @pippinsplugins ?

**Comment by pippinsplugins** [comment]
I'm not sure what the UI would look like but I rather like the idea of being in core.

**Comment by katalystkorner** [comment]
In CAMS, there is an option to do this. To select more than one day, you just hold down the ctrl button. Here is a screenshot of the section:
![ui possibility](https://cloud.githubusercontent.com/assets/19507901/15477417/140c868a-20da-11e6-8f87-cd65c109bf6a.png)
I don't know how much of an addition this would be to the code, but I know it would probably change the relationship type of the file > options.

**Comment by cklosowski** [comment]
Yeah I' not a fan of the multiselect HTML element mostly b/c its horrible on Mobile. We'll work something up for this though.

**Comment by katalystkorner** [comment]
Oh agreed on the mobile!! Thank you for looking into this.

**Comment by chriscct7** [comment]
A better ui interface might be Select2's horizontal input multiselect
